src/gar.lua

```diff
@@ -166,7 +173,10 @@ local function processInstantWithdrawal(qty, elapsedTime, totalWithdrawalTime, f
 	penaltyRate = math.max(
 		constants.MIN_EXPEDITED_WITHDRAWAL_PENALTY_RATE,
 		math.min(constants.MAX_EXPEDITED_WITHDRAWAL_PENALTY_RATE, penaltyRate)
-	) -- Ensure penalty is within bounds
+	)
+
+	-- round to three decimal places
```

Comment:
What's the thought process/spec here? Also might make sense to write a function and unit test for this.

Reply:
the penalty rate does not need precision greater than 3 digits as we round down the resulting mIO charged when a penalty occurs. avoids having to cast to string and potential loss of precision in tests.

Reply:
moved to a util here - [8f13aa5](https://github.com/ar-io/ar-io-network-process/pull/161/commits/8f13aa515d4798eeb8fe2920e20814ba6df7b6d9)

added test here -